test/board.spec.js

```diff
@@ -0,0 +1,14 @@
+const demand = require('must');
+const {createBoard, size} = require('../src/board');
```

Comment:
Here's the other side of exporting: in this case we're _importing_ some of the stuff that the `board` module exports.

See if you can figure out how it works. You'll be doing a _lot_ of exporting and importing so it's a key feature to grasp.